### ISSUE TITLE
paper code？

### ISSUE THREAD

**Issue by DongleUCAS** [tenzi1995/AE-for-Unmixing#1]
Hello, I am very interested in your code. Is this the code of some paper? If so, could you upload the paper? or email it to me: 1184243910@qq.com.
Thank you very much!

**Comment by tenzi1995** [comment]
Yes. The article is still being edited. Please wait for a while : )

**Comment by DongleUCAS** [comment]
Ok. If the article has been edited or can be uploaded, please pass it to me. Thank you!


Best regards,


Le&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Ziqiang Hua"<notifications@github.com&gt;;
发送时间:&nbsp;2019年12月18日(星期三) 中午12:23
收件人:&nbsp;"tenzi1995/AAS"<AAS@noreply.github.com&gt;;
抄送:&nbsp;"董乐"<1184243910@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [tenzi1995/AAS] paper code？ (#1)




Yes. The article is still being edited. Please wait for a while : )

**Comment by tenzi1995** [comment]
The paper is now available on https://ieeexplore.ieee.org/document/9138438

**Comment by DongleUCAS** [comment]
Thank you very much.


Best regards.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "tenzi1995/AAS"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年8月2日(星期天) 晚上9:34
收件人:&nbsp;"tenzi1995/AAS"<AAS@noreply.github.com&gt;;
抄送:&nbsp;"董乐"<1184243910@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [tenzi1995/AAS] paper code？ (#1)





 
The paper is now available on https://ieeexplore.ieee.org/document/9138438
